app/scripts/metamask-controller.js

```diff
@@ -1164,6 +1164,10 @@ export default class MetamaskController extends EventEmitter {
               ),
             );
           },
+          findNetworkClientIdByChainId:
+            this.networkController.findNetworkClientIdByChainId.bind(
+              this.networkController,
+            ),
```

Comment:
is this needed? it was already a hook. this gets passed to the network controller which i dont think takes this